Packs/GuardiCore/Integrations/GuardiCoreV2/GuardiCoreV2.py

```diff
@@ -0,0 +1,448 @@
+    return jwt_token.get("exp")
+
+
+def calculate_fetch_start_time(last_fetch: Optional[str],
+                               first_fetch: Optional[str]):
+    # Taken from the MicrosoftCloudAppSecurity Integration
+    if last_fetch is None:
+        if not first_fetch:
+            first_fetch = '3 days'
+        first_fetch_dt = parse(first_fetch).replace(tzinfo=utc)  # type:ignore
+        # Changing 10-digits timestamp to 13-digits by padding with zeroes,
+        # since API supports 13-digits
+        first_fetch_time = int(first_fetch_dt.timestamp()) * 1000
+        return first_fetch_time
+    else:
+        return int(last_fetch)
+
+
+def filter_human_readable(results: dict, human_columns=List[str]) -> dict:
+    # Takes in results dict and filters out relevant human_columns.
+    filtered = {}
+    for hc in human_columns:
+        if hc in results:
+            filtered[hc] = results[hc]
+    return filtered
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    message: str = ''
+    try:
+        from_time = int(
+            parse("1 days").replace(tzinfo=utc).timestamp()) * 1000
+        to_time = int(
+            parse("now").replace(tzinfo=utc).timestamp()) * 1000
+        client.get_incidents({"from_time": from_time, "to_time": to_time})
+        message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(
+                e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+def get_incidents(client: Client, args: Dict[str, Any]):
+    from_time = args.get('from_time', None)
+    to_time = args.get('to_time', None)
+
+    if not from_time or not to_time:
+        raise DemistoException(
+            "GuardiCoreV2 - get incidents needs from_time and to_time.")
+
+    # Convert time format to epoch
+    from_time = date_to_timestamp(from_time, DATE_FORMAT)
+    to_time = date_to_timestamp(to_time, DATE_FORMAT)
+
+    limit = int(args.get('limit', 50))
+    offset = int(args.get('offset', 0))
+    severity = args.get('severity', [])
+    source = args.get('source', None)
+    destination = args.get('destination', None)
+    tag = args.get('tag', None)
+    incident_type = args.get('incident_type', [])
+
+    result = client.get_incidents({
+        "from_time": from_time,  # Epoch with ms
+        "to_time": to_time,
+        "limit": limit,
+        "offset": offset,
+        "severity": severity,
+        "source": source,
+        "destination": destination,
+        "tag": tag,
+        "incident_type": incident_type
+    })
+
+    raw_results = result.get("objects")
+
+    results = [filter_human_readable(res, human_columns=INCIDENT_COLUMNS) for
+               res in raw_results]
+
+    md = tableToMarkdown(f'GuardiCoreV2 - Incidents: {len(results)}', results)
+
+    return CommandResults(
+        outputs_prefix='Guardicore.Incident',
```

Comment:
You can have another global var for context, e.g:
INTEGRATION_CONTEXT_NAME = 'Guardicore'

Reply:
we also usually have a key to identify on. meaning in your case, that the results in the context will be merged if the incident is on the same id.
see: https://github.com/demisto/content/blob/2b388d5f8d074485ac7c1a592a1a45374d8f7971/Packs/Zimperium/Integrations/Zimperium/Zimperium.py#L601